packages/map/.npmignore

```diff
@@ -0,0 +1,11 @@
+.storybook/
```

Comment:
https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d

using package.json `files` may be simpler?